Sources/Request/Request.swift

```diff
@@ -0,0 +1,27 @@
+public struct Request {
+
+     public let method: String
+     public let path: String
+     public var queries: [String: String] = ["fname": "World"]
+     public var body: [String: String] = [String(): String()]
+     public var cookie: String = String()  
+
+     public init(method: String, path: String, body: [String: String]) {
+         self.method = method
+         self.path = path
+         self.body = body
+     }
+
+     public init(method: String, path: String, queries: [String: String]) {
+         self.method = method
+         self.path = path
+         self.queries = queries
+     }
+
+     public init(method: String, path: String, cookie: String) {
+         self.method = method
+         self.path = path
+         self.cookie = cookie 
+     }
+
```

Comment:
We may have discussed these `init` methods before, but I want to touch on them again. They're all slightly different in the parts of the `Request` that they initialize. Is there a way to consolidate to one that specifies default values for the parts that might be absent? For example,

```swift
public init(method: String, path: String, body: [String: String] = [:], queries: [String: String] = [:], cookie: String) {
  ...
}
```